CONTRIBUTORS.rst

```diff
@@ -0,0 +1,7 @@
+Contributors
+============
+
+Contributors in order of appearance:
+
+* ???
+* Markus Heiser <markus.heiser@darmarIT.de>
```

Comment:
I guess this means you're the first? Welcome! :grin:

Feel free to remove the "???" line from this file, I won't add anyone else's name without their explicit opt-in.

Reply:
Hm .. I have a doubt, should my name really the first on the list? I think my contribution is the smallest of all. Most of the work is done by you @rfk  and @tarekziade. Is it OK if  I add your name as first? @tarekziade is it OK to add your name to the CONTRIBUTORS.rst as second. If so, it might be OK for me to add my Name as third.

Reply:
Sure, this sounds good; given Tarek and I both got paid to work on this thing, I think it'd be find to put both our names in their without getting an explicit OK from @tarekziade.  Thanks!